dat/RecipesScraper/RecipesScraper/spiders/allrecipes_spider.py

```diff
@@ -0,0 +1,55 @@
+"""
+Scrape data from allrecipes.com
+"""
+import re
+import scrapy
+from RecipesScraper.items import RecipeItem
+
+
+class AllRecipesSpider(scrapy.Spider):
+    """Spider to scrape All Recipes (allrecipes.com)"""
+    name = "allrecipes"
+
+    with open("sitemap/seed_lists/allrecipes_seed_list.txt") as seedfile:
+        start_urls = ["{}?page=1".format(url.strip()) for url in seedfile.readlines()]
+
+    def parse(self, response):
```

Comment:
If you base this off of `CrawlSpider` then you can get rid of a lot of boilerplate looking through these sites https://doc.scrapy.org/en/latest/topics/spiders.html#crawlspider-example